app/jobs.py

```diff
@@ -20,11 +21,12 @@
 STATUSES_EMAIL_TEMPLATE = "email_templates/email_request_status_changed"
 
 
-@celery.task(autoretry_for=(OperationalError,), retry_kwargs={'max_retries': 5}, retry_backoff=True)
+@celery.task(autoretry_for=(OperationalError, SQLAlchemyError,), retry_kwargs={'max_retries': 5}, retry_backoff=True)
```

Comment:
Should we add StatementError to `autoretry_for`?

Reply:
`StatementError` is a child of `SQLAlchemyError`. It should be caught by this.